oak_sev_guest/src/instructions.rs

```diff
@@ -147,6 +148,22 @@ impl From<RmpPermission> for u64 {
     }
 }
 
+impl TryInto<RmpPermission> for u64 {
+    type Error = &'static str;
+
+    fn try_into(self) -> Result<RmpPermission, Self::Error> {
+        let bytes = self.to_le_bytes();
+
+        Ok(RmpPermission {
+            target_vmpl: bytes[0],
+            perm_mask: PermissionMask::from_bits(bytes[1]).expect("invalid permission mask value"),
```

Comment:
This does not look like it needs to be "try_into" since it will panic on any failures and no code path ever returns an error.

Reply:
Ah yes, good point -- I was meaning to use `.ok_or("")?` here, as I don't think this function should panic.